apps/cli-daemon/src/app/utils/extract -value-object.ts

```diff
@@ -0,0 +1,49 @@
+import { JsonObject, JsonValue } from '@angular-devkit/core/src/json/utils';
+
+interface JsonObjectNested {
+  [prop: string]: JsonValue | JsonObject | JsonObjectNested | undefined;
+}
+export const extractValueObject = <T>(
```

Comment:
also for the utils, can we work with the same function Angular working with?

Reply:
this function traverse on an object and not only on JSONSchema7.
it it pulling the values from the object. there for i have typing issues for JsonObject  and JSONSchema7

Reply:
@amanecer2levi so it's the same function Agnular uses but the types do not work? or does it does something else than Angular's?

I really want no to add our specific logic so if it's the types case let's look together and find some solution

Reply:
this function extracting data from a json. but i need to cast it back to bee JSONSchema7 (it's a define object)
i will try using https://github.com/sagold/json-schema-library#getchildschemaselection library for the json extraction

i saw this library from the official jsonSchema 
https://json-schema.org/implementations.html#utilities